Software/flight_computer_program/StateMachine.cpp

```diff
@@ -0,0 +1,80 @@
+#include "SensorData.cpp"
+//state machine
+class StateMachine{
+private:
+    SensorData sensor_data;
+public:
+    void set_variables(SensorData data){
+        sensor_data = data;
+    }
+
+    enum FlightStates{
```

Comment:
keep typedefs separate from class definitions

Reply:
I don't quite understand what you mean here?

Reply:
FlightStates is a type definition correct? It defines type of enum, you usually put those in a separate header `.h` or `.hpp` file. So that they can be used by other classes